build.sbt

```diff
@@ -91,17 +151,19 @@ lazy val sampleSlickTestkitExampleProject =
 /* A command that runs all tests sequentially */
 def testAll = Command.command("testAll") { state =>
   val extracted = Project.extract(state)
+
+  // 0.13.8; use Def.sequential?
   val tasks = List(
-    test in (slickTestkitProject, Test),
-    test in (slickTestkitProject, DocTest),
-    test in (osgiTestProject, Test),
-    test in (reactiveStreamsTestProject, Test),
-    packageDoc in Compile in slickProject,
-    packageDoc in Compile in slickCodegenProject,
-    packageDoc in Compile in slickHikariCPProject,
-    packageDoc in Compile in slickTestkitProject,
-    mimaReportBinaryIssues in Compile in slickProject, // enable for minor versions
-    testSamples in aRootProject
+    slickTestkitProject / Test / test,
+    slickTestkitProject / DocTest / test,
+    osgiTestProject / Test / test,
+    reactiveStreamsTestProject / Test / test,
+    slickProject / Compile / packageDoc,
+    slickCodegenProject / Compile / packageDoc,
+    slickHikariCPProject / Compile / packageDoc,
+    slickTestkitProject / Compile / packageDoc,
+    slickProject / Compile / mimaReportBinaryIssues // enable for minor versions
+//    aRootProject / testSamples
```

Comment:
should we be concerned that this is now commented out?

Reply:
`testSample1,2,3,4` contain the same tests as `testSamples`, `updateSampleHelloSlick` updates settings for `sample-hello-slick` (scalaVersion and others) and testSample1 tests it, same for other 3 samples, these arguments passed to sbt for Travis CI [here: build.sh](https://github.com/alexFrankfurt/slick/blob/update-sbt/admin/build.sh#L34). And to test locally: `updateSampleHelloSlick; root/testSample1`

Reply:
okay, I don't really follow because I don't know the code, but if you're satisfied, I'm satisfied :-)